### PR TITLE
Nomination candidate for bunghi

### PR DESCRIPTION
**Pull request by bunghi**
#### Summary of Changes

Fixes #

### CODE REVIEW

**Comment by mvines** [comment]
Hey there's a couple more places that you need to edit.  Please refer to the template for more: https://github.com/solana-labs/solana/pull/14724/files

File: sdk/src/feature_set.rs [hunk 1]
```diff
@@ -49,6 +49,15 @@ pub mod full_inflation {
...
 
+    pub mod bunghi {
+        pub mod vote {
+            solana_sdk::declare_id!("E9hFUVEz29H8XMXk7ygk7ZpCuEuZQ8DJvJKJSTGu1RM6");
+        }
+        pub mod enable {
+            solana_sdk::declare_id!("5S9JDUb4vKY1CUxLf5oc96ZxjGrephj1jcPeTi62sYmP");
+        }
+    }
+
```
Comment (mvines) [inline sdk/src/feature_set.rs:60]: the code prettier wants this whitespace removed
```suggestion

```

**Comment by bunghi** [comment]
Should be ok now..

File: sdk/src/feature_set.rs (hunk 1 above)
Comment (mvines) [inline sdk/src/feature_set.rs:60]: hmm, it's still unhappy.  This is the change it wants I think:
```suggestion

```

Comment (mvines) [reply sdk/src/feature_set.rs:60]: @bunghi - I'll make the change here, don't push a new commit.  Let's see if this works...

**Pull request merged**